frontend/src/components/billing/components/replenish-cards-list/components/replenish-card/replenish-card.tsx

```diff
@@ -0,0 +1,35 @@
+import { PaymentUnit } from 'common/enums/enums';
+import { FC } from 'common/types/types';
+import StripeCheckout, { Token } from 'react-stripe-checkout';
```

Comment:
Maybe it's better to move it to a separate component

Reply:
In that case there will be redundant wrapper with same parameters as StripeCheckout. We need coaches assistance here

Reply:
Then you can remove this wrapper :thinking:

Reply:
Then what is the purpose of a separate component? 🤔 :)

Reply:
Misunderstood the purpose of this component, a separate component can be skipped, but I don't like the fact of importing from `react-stripe-checkout` in 2 different components